### ISSUE TITLE
Wrong version number on top of 3.3.4-rc1 distribution

### ISSUE THREAD

**Issue by sacredseven** [morteza/bootstrap-rtl#80]
At the beginning of 3.3.4-rc1 distribution it states
```
/*******************************************************************************
 *              bootstrap-rtl (version 3.3.2-rc1)
 *      Author: Morteza Ansarinia (http://github.com/morteza)
 *  Created on: March 05,2015
 *     Project: bootstrap-rtl
 *   Copyright: Unlicensed Public Domain
 *******************************************************************************/
```
which is obviously wrong.

**Comment by Amomo** [comment]
Also in 3.3.2, the version info still point to 3.3.2-rc1
```
/*******************************************************************************
 *              bootstrap-rtl (version 3.3.2-rc1)
 *      Author: Morteza Ansarinia (http://github.com/morteza)
 *  Created on: March 05,2015
 *     Project: bootstrap-rtl
 *   Copyright: Unlicensed Public Domain
 *******************************************************************************/
```

**Comment by morteza** [comment]
Fixed in 2efe03782
